bugzoo/mgr/coverage.py

```diff
@@ -77,6 +80,7 @@ def resolve_path(fn_rel: str) -> str:
         files_to_lines = {}
         packages = root.find('packages')
 
+        logger.debug("Starting to traverse all files reported by gcovr. Seconds passed: {}".format(timer() - t_start))
```

Comment:
Prefer the following for logger calls:

```
logger.debug("Starting to blah. Seconds passed: %f", timer() - t_start)
``` 

https://docs.python.org/2/howto/logging.html#logging-basic-tutorial

Reply:
Also, it would be awesome if you could report to 2 or 3 decimal places, rather than a dozen or so.